### ISSUE TITLE
Server does not work on Windows

### ISSUE THREAD

**Issue by gitcnd** [TigerVNC/tigervnc#1040]
- The "About" option in the service windows does nothing at all
- None of the other options in the service menu provide any clue whether or not VNC is listening on anything, or if so, what IPs and Ports are in use
- The code fails to adjust firewall rules to accept incoming connections * hence meaning of "not work" in subject line *
- First-run option settings produce a non-specific "file not found" error, and then promptly discard the settings changes we made (port selections, security options etc).

So basically - not only does the server not work, it does nothing at all to help us diagnose why it is not working either, and doesn't bother to adjust the settings it knows are 100% necessary for operation, and has some dangerous problems with important data-loss issues (security settings) when handling exceptions.

DO NOT REPLY.

This issue is NOT an invitation to discuss any of the above.  Either put your effort into fixing the problems, or close this page.
